astilectron-bundler/main.go

```diff
@@ -7,12 +7,14 @@ import (
 	"path/filepath"
 	"runtime"
 
-	"github.com/asticode/go-astilectron-bundler"
-	"github.com/asticode/go-astilog"
-	"github.com/asticode/go-astitools/flag"
+	astibundler "github.com/asticode/go-astilectron-bundler"
```

Comment:
Could you remove the alias in the imports in both the `main.go` and `bundler.go` files?

Reply:
Yeah sorry, all the other stuff I work on requires the use of goimports which is what did this